pkg/storage/stores/shipper/downloads/index_set.go

```diff
@@ -294,7 +294,7 @@ func (t *indexSet) sync(ctx context.Context, lock bool) (err error) {
 		return err
 	}
 
-	level.Debug(util_log.Logger).Log("msg", fmt.Sprintf("updates for table %s. toDownload: %s, toDelete: %s", t.tableName, toDownload, toDelete))
+	level.Debug(t.logger).Log("msg", fmt.Sprintf("sync updates. toDownload: %s, toDelete: %s", toDownload, toDelete))
```

Comment:
```suggestion
	level.Debug(t.logger).Log("msg", "sync updates", "toDownload", toDownload, "toDelete", toDelete)
```

This will make it easier to aggregate on these fields

Reply:
Also not sure if `sync updates` is the most useful log message here; can we find something more descriptive maybe?

Reply:
yeah, it makes sense. I will update the message.

Reply:
thanks for the review!